helper.py

```diff
@@ -0,0 +1,30 @@
+# Copies files and folders to relevant containers
+import os
+
+
+def cp_helper(name, test_name, container):
+    if test_name == "storage":
+        dest = "scratch/" + name + "/"
+        docker_cp_host(container, dest)
+    elif test_name == "jupyterhub-api":
+        container_name = "jupyterhub"
+        docker_cp_host(container_name, "")
+    else:
+        docker_cp_host(container, "")
+
+
+def docker_cp_host(container_name, dest):
+    file_list = ['run.py', 'test.yaml', 'run_container.py']
+    folder_list = ['tests']
+    for file in file_list:
+        try:
+            cmd = "docker cp %s %s:/%s%s" % (file, container_name, dest, file)
```

Comment:
As per comment on line 7, make sure you already have the full path here instead of building it with random slashes